src/lib/gundb/UserPropertiesClass.js

```diff
@@ -96,4 +96,8 @@ export default class UserProperties {
 
     return true
   }
+
+  reset() {
+    return this.gun.secret(UserProperties.defaultProperties)
```

Comment:
```suggestion
    const { defaultProperties } = UserProperties

    this.data = Object.assign({}, defaultProperties)
    return this.gun.secret(defaultProperties)
```

Reply:
and pls merge https://github.com/GoodDollar/GoodDAPP/pull/2243/files once it will be merged to the master